0800-appendix-195943-stock.Rmd

```diff
@@ -175,39 +144,29 @@ fpr::fpr_table_cv_summary_memo(dat = pscis_phase1, site = 1805507, site_photo_id
 ## Stream Characteristics Upstream of `r as.character(my_site)` and Downstream of `r as.character(my_site2)` {-}
 
 `r fpr_my_survey_dist(loc = 'us')`
-`r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-195943-03) - \\@ref(fig:photo-195943-04)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-195943-d02))."))`
-`r fpr_my_habitat_paragraph(loc = 'us')` The first \~250m of channel
-upstream of the highway were noted as beaver influenced wetland with
-extensive dry sections at the time of the survey. Frequent pockets of
-gravels and small cobbles suitable for rainbow trout and coho salmon
-spawning were noted throughout the area surveyed. A canyon section of
-stream was located upstream of the crossing approximately 300m contained
-deep bedrock pools with small fish present and no large woody debris.
-One deep pool upstream of the canyon section contained likely 2-3 year
-old juveniles (coloring indicated rainbow trout, unverified). The
+`r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-195943-03) - \\@ref(fig:photo-195943-04)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-195943-d02))."))`  
+`r fpr_my_habitat_paragraph(loc = 'us')` There was good flow volume and a decent amount of complexity due to boulders and undercut banks. There were small pockets of gravel that may be suitable for resident rainbow and coho spawning. Occasional shallow pools and traces of large water debris were sighted. The channel appeared to be constricted due to agricultural development on both sides of the stream. Water withdrawal is occurring as well. More agricultural development was encroaching on the riparian farther upstream. The
```

Comment:
we know that the gravels are suitable.... 

There were small pockets of gravel suitable for resident rainbow as well as steelhead and coho spawning